src/bot_worker/celery.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import absolute_import
+from celery import Celery
+
+worker = Celery('iwant_bot_celery_worker',
+                broker='redis://redis:6379/0',
+                backend='redis://redis:6379/0'
+               )
+
+worker.conf.update(
+                    enable_utc=True,
+                    broker_use_ssl=False,
+                    timezone='Europe/Prague',
+                    worker_concurrency=6,
+                    imports=['bot_worker.tasks', 'iwant_bot.pool'],
+                  )
+worker.conf.beat_schedule = {
```

Comment:
it is possible to add some task to be scheduled, clean up of database on 1st day of every month for example.